DOC-436 auth-session reference (#14699)
cockroachdb/docs: v21.1/cockroach-auth-session.md

### `auth-session logout`

Field | Description
------|------------
`username` | The username of the user whose session was revoked.
`session ID` | The session ID to the HTTP interface previously established for that user.
`revoked` | The date and time of revocation for that user's authenticated session.

### `auth-session list`

Field | Description
------|------------
`username` | The username of the user authenticated.
`session ID` | The session ID to the HTTP interface established for that user.
`created` | The date and time a session was created.
`expired` | The date and time a session expired.
`revoked` | The date and time of revocation for that user's authenticated session. If the session is still active, this will appear as `NULL`.
`last used` | The date and time of the last access to the HTTP interface using this session token.

## Required roles

To run any of the `auth-session` subcommands, you must be a member of the [`admin` role](authorization.html#admin-role). The user being authenticated via `login` or `logout` does not require any special roles.

## Considerations

> - The `login` subcommand allows administrative users to create HTTP authentication tokens with an arbitrary expiration date and time. If operational policy requires stricter control of authentication sessions, you can:

Reviewer: I'd just say `admin` in codes rather than "administrative", referring to the admin role as specified earlier in the doc. More exact as well as a shorter word. This applies in all cases throughout the PR.
In your example, looks like you specify duration, not expiration date/time.
Also, I suggest breaking things out a bit.
```suggestion
- The `login` subcommand allows `admin` users to create HTTP authentication tokens with arbitrary duration.
    
    {{site.data.alerts.callout_info}}
    If operational policy requires stricter control of authentication sessions, you can:
```

>   - Monitor `system.web_sessions` for all current and recent HTTP sessions.

Reviewer: Not sure if the indentation and all will work out here, trying to do this in the browers...
```suggestion
        - Monitor `system.web_sessions` for all current and recent HTTP sessions.
```

>   - Revoke HTTP authentication tokens as needed with the `logout` subcommand.

Reviewer: ```suggestion
      - Revoke HTTP authentication tokens as needed with the `logout` subcommand.
```

>   - Set the `--expire-after` flag with a finer granularity.

Reviewer: ```suggestion
      - Set the `--expire-after` flag with a shorter duration.
      {{site.data.alerts.end}}

```

> - The `logout` subcommand logs out all sessions for the given user; you cannot target individual sessions for logout. If more control is desired, consider setting the `--expire-after` flag with a finer granularity.

Reviewer: ```suggestion
    - The `logout` subcommand logs out all sessions for the given user; you cannot target individual sessions for logout.
    
    {{site.data.alerts.callout_info}}
    If more control is desired, consider setting the `--expire-after` flag with a finer granularity.
    {{site.data.alerts.end}}
```
